mmdet/datasets/pipelines/transforms.py

```diff
@@ -72,6 +76,7 @@ def __init__(self,
             # mode 2: given multiple scales or a range of scales
             assert multiscale_mode in ['value', 'range']
 
+        self.backend = backend
```

Comment:
We need to add an assertion here to check the backend is in `['cv2', 'pillow']`.

Reply:
I think it might be better to just leave it as it is here, snce it has been correctly checked in `mmcv.imresize` function. It also allows future extension of adding new imresize backends by just modifying codes in mmcv.